packages/jaeger-ui/src/model/ddg/GraphModel.tsx

```diff
@@ -20,15 +20,15 @@ import { decode } from './visibility-codec';
 
 import { PathElem, TDdgDistanceToPathElems, TDdgModel, TDdgVertex } from './types';
 
-export default class Graph {
+export default class GraphModel {
   private distanceToPathElems: TDdgDistanceToPathElems;
   private pathElemToEdge: Map<PathElem, TEdge>;
   private pathElemToVertex: Map<PathElem, TDdgVertex>;
   private vertexToPathElems: Map<TDdgVertex, Set<PathElem>>;
   private vertices: Map<string, TDdgVertex>;
   private visIdxToPathElem: PathElem[];
 
-  constructor({ ddgModel }: { ddgModel: TDdgModel }) {
+  constructor(ddgModel: TDdgModel) {
```

Comment:
this change will have conflicts with my PRs that add more data to the constructor